datanommer.commands/datanommer/commands/__init__.py

```diff
@@ -304,3 +305,24 @@ def formatter(key, val):
         results.append(formatter(result.category, result))
 
     click.echo(f"[{','.join(results)}]")
+
+
+@click.command()
+@config_option
+def refresh_view(config_path):
+    """Refresh the materialized view `recent_topics`.
+
+    This command should be run periodically via cron job to keep
+    the materialized view `recent_topics` up to date.
+    """
+    config = get_config(config_path)
+    m.init(
+        config["datanommer_sqlalchemy_url"],
+        alembic_ini=config["alembic_ini"],
+    )
+
+    # Get database connection and refresh the view
+    with m.session.get_bind().connect() as connection:
+        refresh_recent_topics(connection)
```

Comment:
You can use the session directly as the argument:
```suggestion
    refresh_recent_topics(m.session)
```